app/grandchallenge/workstation_configs/models.py

```diff
@@ -221,6 +222,14 @@ class ImageInterpolationType(models.TextChoices):
         help_text="The method used to interpolate multiple voxels of overlay images and project them to screen pixels",
     )
 
+    default_interpolation = models.CharField(
```

Comment:
This won't be applied to Overlays, right? Could we call it `default_image_interpolation`?

Reply:
Done